apps/fluent-tester/src/TestComponents/FocusZone/styles.ts

```diff
@@ -47,14 +47,16 @@ export const focusZoneTestStyles = StyleSheet.create({
   smallBoxStyle: {
     width: 20,
     height: 20,
-    backgroundColor: 'lightgrey',
     margin: 5,
+    // something is wrong with setting the background color on this style on the Mac, and until we fix that, let's not do this
+    ...Platform.select({ macos: { borderWidth: 1 }, default: { backgroundColor: 'lightgrey' } }),
```

Comment:
Could you elaborate?

Reply:
![image](https://github.com/user-attachments/assets/e6f13f78-240e-43df-b8fc-25e2f893e518)

This is all I see, there's nothing visible in the gray area above. However, the controls DO exist, they can be focused, and I bet you could hear them via VO.

Reply:
@Saadnajmi this is the issue with the Text backgroundColor support from https://github.com/microsoft/fluentui-react-native/blob/4d1c92173633ca23fc153890fda6472e732e9e50/apps/fluent-tester/src/TestComponents/FocusZone/FocusZoneTest.tsx#L131C1-L145C3
```
const TinyText = () => {
  return (
    <Text selectable style={focusZoneTestStyles.smallBoxStyle}>
      Hi
    </Text>
  );
};

const WideText = () => {
  return (
    <Text selectable style={focusZoneTestStyles.wideBoxStyle}>
      Hello world
    </Text>
  );
};
```